source/extensions/transport_sockets/alts/config.cc

```diff
@@ -84,8 +100,9 @@ createTransportSocketFactoryHelper(const Protobuf::Message& message, bool is_ups
     tsi_handshaker* handshaker = nullptr;
     // Specifying target name as empty since TSI won't take care of validating peer identity
     // in this use case. The validation will be performed by TsiSocket with the validator.
-    tsi_result status = alts_tsi_handshaker_create(
-        options.get(), target_name, handshaker_service.c_str(), is_upstream, &handshaker);
+    tsi_result status =
+        alts_tsi_handshaker_create(options.get(), target_name, handshaker_service.c_str(),
+                                   is_upstream, nullptr /* interested_parties */, &handshaker);
```

Comment:
I had a TODO here to follow up on if Envoy would benefit from allowing this pollset to be configurable. Do you think this would be useful?

Reply:
I'll defer to @JimmyCYJ on this one.